packages/source-maps/readme.md

```diff
@@ -0,0 +1,154 @@
+# @zodern/source-maps
```

Comment:
I'm curious why you are forking my npm package? I would prefer if you use the npm package instead, though forking is of course okay. I would ask that you also copy over the license file: https://github.com/zodern/source-maps/blob/main/LICENSE. I realized it was missing one.

Reply:
It's just a POC, isn't the final implementation. We still need to discuss the best approach to internalize community packages(will do monday), so don't worry; we will definitely change this implementation ;)